src/bulk/common.ts

```diff
@@ -1268,6 +1264,18 @@ export abstract class BulkOperationBase {
     batchType: BatchType,
     document: Document | UpdateStatement | DeleteStatement
   ): this;
+
+  private shouldForceServerObjectId(): boolean {
+    if (typeof this.s.options.forceServerObjectId === 'boolean') {
+      return this.s.options.forceServerObjectId;
+    }
+
+    if (typeof this.s.collection.s.db.options?.forceServerObjectId === 'boolean') {
+      return this.s.collection.s.db.options?.forceServerObjectId;
+    }
+
+    return false;
```

Comment:
```suggestion
    return this.s.options.forceServerObjectId === true || this.s.collection.s.db.options.forceServerObjectId === true;
```

Either option is either set precisely to `true` or we default to `false`, just a suggestion

Reply:
good catch! fixed.